management-microservice/src/main/resources/application.yml

```diff
@@ -0,0 +1,10 @@
+spring:
+  datasource:
+    url: jdbc:mysql://localhost/skt
+    username: root
```

Comment:
don't store usernames or passwords in your repository ... you can use placeholders instead